.github/workflows/maven.yml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build samples
+      run: mvn install --file fhir-examples/pom.xml
+    - name: Build with Maven
+      run: mvn package --file fhir-parent/pom.xml
```

Comment:
I think we should build fhir-tools as well? thoughts?

Reply:
I was actually looking forward to removing the step that builds fhir-examples to speed it up a tad.  I guess it boils down to what we're hoping to achieve with this...ideally there would be some way to only build the maven projects that actually have changes (either directly or in their dependencies).
However, since I don't know how to do that, I'm not opposed to building "everything" for now.  Should we build the fhir-benchmark project while we're at it, too?  Know of any others that aren't hooked into the main server build?

Reply:
OK, I pushed a change to build everything (including fhir-tools, fhir-examples-generator, and fhir-benchmark.